python/build-requirements.txt

```diff
@@ -12,6 +12,7 @@ mypy
 tensorflow==2.0.4
 numpy==1.18.5
 pandas==1.2.1
+scipy==1.3.2
```

Comment:
@mohazahran I don't see this version as an approved scipy version internally. Any reason you need this specifically? Will slack you the link to internal PyPi.

Reply:
Good catch. I didn't check it against that list. I will update it.